dspace/config/modules/actuator.cfg

```diff
@@ -0,0 +1,57 @@
+
+#---------------------------------------------------------------#
+#------------SPRING BOOT ACTUATOR CONFIGURATION-----------------#
+#---------------------------------------------------------------#
+
+# Health endpoint configuration, for more details see https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints.health
+
+## Configuration to establish when show the health status details
+management.endpoint.health.show-details = when-authorized
+## Configuration which users can see the health status details
+management.endpoint.health.roles = ADMIN
+## Configuration to establis
+management.endpoint.health.status.order= down, out-of-service, up-with-issues, up, unknown
+## Configuration that enables only health and info endpoints
+management.endpoints.web.exposure.include=health,info
+
+## Configuration to set 200 as status of health http response when it is DOWN or OUT_OF_SERVICE
+management.endpoint.health.status.http-mapping.down = 200
+management.endpoint.health.status.http-mapping.out-of-service = 200
```

Comment:
I'd recommend we add a note here to say:
"The DSpace UI requires these be set to 200 in order to support health status reports when services are down."